pkg/history/time/time.go

```diff
@@ -0,0 +1,34 @@
+package time
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fidelity/kconnect/pkg/aws/awsconfig"
+	"gopkg.in/ini.v1"
+)
+
+func GetExpireTimeFromAWSCredentials(profileName string) (time.Time, error) {
+
+	path, err := awsconfig.LocateConfigFile()
```

Comment:
I haven't tried but i wonder if this could be simplified using `credentials.NewSharedCredentials("", "kconnect-2338640006")` and then using the `ExpiresAt`....not tried this though so it may be a red herring

Reply:
I did try it this way first and it kept returning an error, "expire time not available" or something along those lines

Reply:
I thought you would've tried it :)